js/index.js

```diff
@@ -0,0 +1,136 @@
+// Array of card data
+const cardData = [
+  {
+    image: '../images/speaker/speaker-1.png',
+    text: 'Yochai Benkler',
```

Comment:
[OPTIONAL]

Hi, I noticed that your object keys are not very descriptive and consistent. For example, `image` could be renamed to `photo`, and `text` could be renamed to `name`. This will make your code more clear and easy to understand. I suggest you use more descriptive and specific names for your object keys, such as:

{ name: "Yochai Benkler",  role: "Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School",  description: "Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006",  photo: "../images/speaker/speaker-1.png"}

This will also help you avoid confusion when you have multiple objects with similar keys. Hope this helps! 😊